### PR TITLE
Updated metadata

### PR DESCRIPTION
**Pull request by vijay8451**
- Updated tests metadata.
- testimony results:
```
(testimony) [root@vijsingh testimony]# testimony -c ../PR_Proj/robottelo/testimony.yaml validate ../PR_Proj/robottelo/tests/foreman/api/test_repository.py 
Total number of tests: 86
Total number of invalid docstrings: 0 (00.00%)
Test cases with no docstrings: 0 (0.00%)
Test cases missing minimal docstrings: 0 (0.00%)
Test cases with unexpected tags: 0 (0.00%)
Test cases with unexpected token values in docstrings: 0 (0.00%)
Test cases with unparseable docstrings: 0 (0.00%)
(testimony) [root@vijsingh testimony]# testimony -c ../PR_Proj/robottelo/testimony.yaml validate ../PR_Proj/robottelo/tests/foreman/api/test_repository_set.py 
Total number of tests: 2
Total number of invalid docstrings: 0 (00.00%)
Test cases with no docstrings: 0 (0.00%)
Test cases missing minimal docstrings: 0 (0.00%)
Test cases with unexpected tags: 0 (0.00%)
Test cases with unexpected token values in docstrings: 0 (0.00%)
Test cases with unparseable docstrings: 0 (0.00%)
(testimony) [root@vijsingh testimony]# testimony -c ../PR_Proj/robottelo/testimony.yaml validate ../PR_Proj/robottelo/tests/foreman/cli/test_repository.py 
Total number of tests: 94
Total number of invalid docstrings: 0 (00.00%)
Test cases with no docstrings: 0 (0.00%)
Test cases missing minimal docstrings: 0 (0.00%)
Test cases with unexpected tags: 0 (0.00%)
Test cases with unexpected token values in docstrings: 0 (0.00%)
Test cases with unparseable docstrings: 0 (0.00%)
(testimony) [root@vijsingh testimony]# testimony -c ../PR_Proj/robottelo/testimony.yaml validate ../PR_Proj/robottelo/tests/foreman/cli/test_repository_set.py 
Total number of tests: 7
Total number of invalid docstrings: 0 (00.00%)
Test cases with no docstrings: 0 (0.00%)
Test cases missing minimal docstrings: 0 (0.00%)
Test cases with unexpected tags: 0 (0.00%)
Test cases with unexpected token values in docstrings: 0 (0.00%)
Test cases with unparseable docstrings: 0 (0.00%)
(testimony) [root@vijsingh testimony]# testimony -c ../PR_Proj/robottelo/testimony.yaml validate ../PR_Proj/robottelo/tests/foreman/ui_airgun/test_repository.py 
Total number of tests: 20
Total number of invalid docstrings: 0 (00.00%)
Test cases with no docstrings: 0 (0.00%)
Test cases missing minimal docstrings: 0 (0.00%)
Test cases with unexpected tags: 0 (0.00%)
Test cases with unexpected token values in docstrings: 0 (0.00%)
Test cases with unparseable docstrings: 0 (0.00%)
(testimony) [root@vijsingh testimony]# testimony -c ../PR_Proj/robottelo/testimony.yaml validate ../PR_Proj/robottelo/tests/foreman/api/test_gpgkey.py 
Total number of tests: 12
Total number of invalid docstrings: 0 (00.00%)
Test cases with no docstrings: 0 (0.00%)
Test cases missing minimal docstrings: 0 (0.00%)
Test cases with unexpected tags: 0 (0.00%)
Test cases with unexpected token values in docstrings: 0 (0.00%)
Test cases with unparseable docstrings: 0 (0.00%)
(testimony) [root@vijsingh testimony]# testimony -c ../PR_Proj/robottelo/testimony.yaml validate ../PR_Proj/robottelo/tests/foreman/cli/test_gpgkey.py 
Total number of tests: 37
Total number of invalid docstrings: 0 (00.00%)
Test cases with no docstrings: 0 (0.00%)
Test cases missing minimal docstrings: 0 (0.00%)
Test cases with unexpected tags: 0 (0.00%)
Test cases with unexpected token values in docstrings: 0 (0.00%)
Test cases with unparseable docstrings: 0 (0.00%)
(testimony) [root@vijsingh testimony]# testimony -c ../PR_Proj/robottelo/testimony.yaml validate ../PR_Proj/robottelo/tests/foreman/cli/test_ping.py 
Total number of tests: 1
Total number of invalid docstrings: 0 (00.00%)
Test cases with no docstrings: 0 (0.00%)
Test cases missing minimal docstrings: 0 (0.00%)
Test cases with unexpected tags: 0 (0.00%)
Test cases with unexpected token values in docstrings: 0 (0.00%)
Test cases with unparseable docstrings: 0 (0.00%)
(testimony) [root@vijsingh testimony]# 

```

### CODE REVIEW

File: tests/foreman/api/test_repository_set.py [hunk 1]
```diff
@@ -9,9 +9,9 @@
 
 :CaseAutomation: Automated
 
-:CaseLevel: Acceptance
+:CaseLevel: Component
 
-:CaseComponent: API
+:CaseComponent: Repositories
```
Comment (omkarkhatavkar) [inline tests/foreman/api/test_repository_set.py]: Not sure if that valid component `:CaseComponent: Repositories` already has cover, don't we have  Repository_Set

@mirzal Is the testimony.yaml is single truth or we can add Components similar like this case?

File: tests/foreman/cli/test_gpgkey.py [hunk 2]
```diff
@@ -1183,7 +1183,7 @@ def test_positive_consume_content_using_repo(self):
 
         :CaseAutomation: notautomated
 
-        :CaseLevel: System
+        :CaseLevel: Integration
```
Comment (omkarkhatavkar) [inline tests/foreman/cli/test_gpgkey.py]: If the `:CaseLevel: Integration` and `::CaseLevel: Component` then would make more sense to change the tier3 to tier2.

File: tests/foreman/cli/test_gpgkey.py [hunk 3]
```diff
@@ -1200,7 +1200,7 @@ def test_positive_consume_content_using_repos(self):
...
         """
```
Comment (omkarkhatavkar) [inline tests/foreman/cli/test_gpgkey.py]: Same as above

File: tests/foreman/cli/test_gpgkey.py [hunk 4]
```diff
@@ -1216,7 +1216,7 @@ def test_positive_consume_content_using_repos_and_different_keys(self):
...
```
Comment (omkarkhatavkar) [inline tests/foreman/cli/test_gpgkey.py]: same as above

File: tests/foreman/api/test_repository_set.py (hunk 1 above)
Comment (vijay8451) [reply tests/foreman/api/test_repository_set.py]: @omkarkhatavkar we do not have any component name on Bugzilla with name `Repository/Repositories Set` it's only `Repositories`

Comment (omkarkhatavkar) [reply tests/foreman/api/test_repository_set.py]: ok then

File: tests/foreman/cli/test_gpgkey.py (hunk 2 above)
Comment (vijay8451) [reply tests/foreman/cli/test_gpgkey.py]: done

File: tests/foreman/cli/test_gpgkey.py (hunk 3 above)
Comment (vijay8451) [reply tests/foreman/cli/test_gpgkey.py]: done

File: tests/foreman/cli/test_gpgkey.py (hunk 4 above)
Comment (vijay8451) [reply tests/foreman/cli/test_gpgkey.py]: done

File: tests/foreman/api/test_repository_set.py (hunk 1 above)
Comment (mirzal) [reply tests/foreman/api/test_repository_set.py]: @omkarkhatavkar Bugzilla is our source of truth. `testimony.yaml` is created automatically by extracting values from Bugzilla. If you see any discrepancy between Bugzilla and `testimony.yaml`, then trust Bugzilla - and let me know, `testimony.yaml` should be fixed.

**Pull request merged**